internal/fftypes/types.go

```diff
@@ -3,7 +3,7 @@
 package fftypes
 
 import (
-	"github.com/GoogleCloudPlatform/functions-framework-go/internal/metadata"
+	"cloud.google.com/go/functions/metadata"
```

Comment:
Does this mean builds with vendored dependencies are going to break when we start using this as the default version in the buildpacks?

Reply:
No, this dependency been in the FF Go for a long time, so it's unlikely any existing vendored dependency functions has a conflict. It was the new dependency on the "cloud.google.com/go/pubsub" package that caused conflicts recently. That package is still omitted.

We don't have too much of a choice to restore this package because some functions today (and our OSS samples) use the `metadata.FromContext` function and are breaking with v1.5.0.